charts/osm/templates/prometheus-configmap.yaml

```diff
@@ -47,7 +51,11 @@ data:
       - job_name: 'kubernetes-pods'
         kubernetes_sd_configs:
         - role: pod
-        relabel_configs:
+        metric_relabel_configs:
+        - source_labels: [__name__]
+          regex: .*bucket.*
```

Comment:
We will need `osm_request_duration_ms_bucket` for SMI metrics eventually. Is there an easy way to ignore all but that one?

Reply:
I think the proper solution might be a ~whitelist~ allowed list for what we explicitly want. Didn't go the extra mile as I'm onto something else, but if we feel we don't want this at all, I will not push for this either.

Reply:
SMI metrics is far enough out that it shouldn't bock this change. Once we get to that point, it should be simple enough to get it to work.